MAINTAINERS.md

```diff
@@ -0,0 +1,20 @@
+Maintainers
+===========
+
+**Active Maintainers**
+
+| Name          | GitHub                 | email                   |
+|---------------|------------------------|-------------------------|
+| Jiyong Huang  | [ngjaying][ngjaying]   | <ngjaying@gmail.com>    |
```

Comment:
It seems this `[ngjaying][ngjaying]` markdown style is not supported on GitHub.

e.g.:
<img width="619" alt="Screenshot 2023-07-12 at 4 26 54 PM" src="https://github.com/lf-edge/ekuiper/assets/28824352/683bebfd-6088-4679-8132-e0c3d13c2bd4">